plugins/miscellanea/nanosound/install.sh

```diff
@@ -5,22 +5,21 @@ sudo apt-get update
 # Install the required packages via apt-get
 
 #START OF LIRC
-sudo apt-get -y install lirc
-echo "Applying LIRC starting policy"
-systemctl disable lirc.service
-systemctl stop lirc.service
-
-echo "Creating lircrc file"
-sudo touch /etc/lirc/lircrc
+#sudo apt-get -y install lirc
+#echo "Applying LIRC starting policy"
+#systemctl disable lirc.service
+#systemctl stop lirc.service
+sudo apt-get -y remove lirc
+
+#echo "Creating lircrc file"
+#touch /etc/lirc/lircrc
 #END OF LIRC
 
 #START OF python devs
-sudo apt-get -y install i2c-tools python-smbus python-pip python-dev python-imaging python-rpi.gpio python-lirc
-
-
-sudo pip install --upgrade setuptools
-sudo easy_install -U pip
-sudo pip install --upgrade python-mpd2 socketIO-client
+sudo apt-get -y install i2c-tools python-smbus python-pip python-dev python-imaging python-rpi.gpio
+sudo apt-get -y remove python-pip
```

Comment:
Why this is necessary? This might compromise the system. Can this be avoided?

Reply:
Isn't there any other solution? This might affect all packages working with python...

Reply:
But the combined results of these 2 lines together below will just do an upgrade of pip, why would it affect?

sudo apt-get -y remove python-pip
 sudo -H easy_install pip

Reply:
Is there any update to this? thanks